orangecontrib/network/widgets/tests/test_OWNxExplorer.py

```diff
@@ -96,6 +96,16 @@ def test_input_subset(self):
         sub_mask = self.widget.get_subset_mask()
         self.assertIsNone(sub_mask)
 
+    def test_clear_selection_on_no_data(self):
+        self.widget.relayout = Mock()
+        self.send_signal(self.widget.Inputs.network, self.network)
+        self.widget.graph.selection_select(np.arange(0, 5))
+        self.assertEqual(self.widget.nSelected, 5)
+        self.assertIsNotNone(self.widget.selection)
+        self.send_signal(self.widget.Inputs.network, None)
+        self.assertEqual(self.widget.nSelected, 0)
+        self.assertIsNone(self.widget.selection)
```

Comment:
I prefer testing a widgets behavior to its properties. This test does not assure no selection on the output.
Adding

`self.assertIsNone(self.get_output(self.widget.Outputs.selected_data))`

results in failed test.